docs/install/install-redis/install-redis-on-linux.md

```diff
@@ -35,7 +35,7 @@ sudo apt-get install redis
 
 ## Install from Snapcraft
```

Comment:
We don't have snapcraft yet. This section isn't accurate.

Reply:
I can remove this section.

Reply:
I think we should.